### DISCUSSION TITLE
Add custom auto-lock time

### DISCUSSION THREAD

**Discussion by patrickReiis** [standardnotes/forum#3645] [Suggestions]
These options are too extreme, terrible UX experience.

![image](https://github.com/standardnotes/forum/assets/76563803/cbadf7ff-2f28-4432-95dd-2ad2736c80a2)

**Comment by thezacharytaylor** [comment]
Seconding this. Over even a few more longer options. =\

**Comment by patrickReiis** [comment]
Any updates? @effieeee

**Comment by patrickReiis** [comment]
This is a life or death situation.

**Comment by patrickReiis** [comment]
Fixed: https://github.com/standardnotes/app/pull/2879
